<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapas</class>
 <widget class="QWidget" name="Mapas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>551</width>
     <height>481</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>5000</number>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_erase">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>240</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 10px;

}
QLabel:hover:!pressed{
	  background-color: #FFFFFF; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/delete.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_home_casa">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 10px;

}
QLabel:hover:!pressed{
	  background-color: #FFFFFF; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/home.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_close">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 10px;

}
QLabel:hover:!pressed{
	  background-color: #FFFFFF; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/close_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #FFFFFF; 
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/app_logo.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelStateAnimated" name="l_max_min">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 10px;

}
QLabel:hover:!pressed{
	  background-color: #FFFFFF; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/max_min_off.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLineEditAutoComplete" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_map_search">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 6px;

}
QLabel:hover:!pressed{
	  background-color: #FFFFFF; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/map_search.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>quickWidget</zorder>
  <zorder>pb_home_casa</zorder>
  <zorder>pb_erase</zorder>
  <zorder>pb_close</zorder>
  <zorder>label</zorder>
  <zorder>l_max_min</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pb_map_search</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelStateAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelstateanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEditAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineeditautocomplete.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
